firefox-ios/Client/Frontend/Browser/Tabs/Views/InactiveTabs/InactiveTabsCell.swift

```diff
@@ -21,9 +20,7 @@ class InactiveTabsCell: UICollectionViewListCell, ReusableCell, ThemeApplicable
     private lazy var bottomSeparatorView: UIView = .build { _ in }
 
     private lazy var titleLabel: UILabel = .build { label in
-        label.font = DefaultDynamicFontHelper.preferredFont(
-            withTextStyle: .caption1,
-            size: UX.titleFontSize)
+        label.font = FXFontStyles.Regular.subheadline.scaledFont()
```

Comment:
need to confirm with @cwzilla on this change

Reply:
Discussed with @cwzilla, can we update this to use `.body` to be in sync with our fonts we used in our list items (i.e. settings, bookmarks)

Reply:
done!

Reply:
awesome, thank you @darkCavalier11 !